packages/expo-updates/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### 🛠 Breaking changes
 
+- Revert [#12734](https://github.com/expo/expo/pull/12734). This introduces an implicit dependency on expo-asset@8.3.3 or above. ([#13733](https://github.com/expo/expo/pull/13733) by [@jkhales](https://github.com/jkhales))
```

Comment:
Nit, but the dependency is actually the other way around now; expo-asset@8.3.3 needs this code to function fully and correctly.

Reply:
Ah OK!